html/modules/custom/iasc_hero/src/Form/HeroRevisionRevertForm.php

```diff
@@ -136,7 +136,7 @@ public function submitForm(array &$form, FormStateInterface $form_state) {
   protected function prepareRevertedRevision(HeroInterface $revision, FormStateInterface $form_state) {
     $revision->setNewRevision();
     $revision->isDefaultRevision(TRUE);
-    $revision->setRevisionCreationTime(REQUEST_TIME);
+    $revision->setRevisionCreationTime(time());
```

Comment:
A bit picky, but the deprecation notice suggests something more verbose: https://www.drupal.org/node/2785211 (and in a few other instances in this module)

Reply:
I know, but injecting a service to get a timestamp felt like a bit of overload